GOVERNANCE.md

```diff
@@ -0,0 +1,126 @@
+Governance for the R data.table project
+
+# Purpose and scope
+
+## This document
+
+The purpose of this document is to define how people related to the project work together, so that the project can expand to handle more contributions, from a larger and more diverse group of contributors.
+
+## The R package
+
+The purpose of the project is to maintain the R data.table package,
+which is guided by the following principles:
+
+* Few (if any) dependencies
```

Comment:
'dependencies' is very overloaded here.

 - System vs. other R package dependencies (we want to minimize both)
 - Enhances/Suggests/Imports/LinkingTo/Depends dependencies (on R packages). I think we want _strictly 0_ Imports/LinkingTo/Depends dependencies (well, maybe we could eventually split `IDateTime` into its own package that we depend on -- so more specifically, _strictly 0 **external** dependencies_, meaning those not maintained by the project). And then few (if any) Suggests/Enhances dependencies.